consent-ws/src/main/java/org/broadinstitute/consent/http/db/ConsentMapper.java

```diff
@@ -13,20 +14,25 @@
 
     public Consent map(int index, ResultSet r, StatementContext ctx) throws SQLException {
         Consent consent = new Consent();
-        consent.consentId = r.getString("consentId");
-        consent.requiresManualReview = r.getBoolean("requiresManualReview");
-        consent.dataUseLetter = r.getString("dataUseLetter");
-        consent.dulName = r.getString("dulName");
+        consent.setConsentId(r.getString("consentId"));
+        consent.setRequiresManualReview(r.getBoolean("requiresManualReview"));
+        consent.setDataUseLetter(r.getString("dataUseLetter"));
+        consent.setDulName(r.getString("dulName"));
         try {
-            consent.useRestriction = UseRestriction.parse(r.getString("useRestriction"));
+            consent.setUseRestriction(UseRestriction.parse(r.getString("useRestriction")));
         } catch (IOException e) {
             throw new SQLException(e);
         }
-        consent.name = r.getString("name");
+        try {
+            consent.setDataUse(DataUseDTO.parseDataUse(r.getString("dataUse")));
+        } catch (IOException e) {
+            throw new SQLException(e);
+        }
```

Comment:
Couldn't the two `try-catch` blocks above be merged into one since they're checking for and throwing the same exceptions? It also made me wonder if there is a more specific `Exception` subclass than `IOException` that could provide more clarity.

Reply:
Great point - they certainly could. As written, we'd lose which statement threw the exception, but I can revise that.